manual/install_instructions

```diff
@@ -1,9 +1,24 @@
+
+get source code
+===============
+
 git clone git@github.ugent.be:hpcugent/mympingpong.git
 
+prepare environment
+===================
+
 module load Python/2.7.9-intel-2015a 
 module load vsc-mympirun/3.4.2
```

Comment:
i have no intentions to maintain this. maybe we should just remove it now EB works?